src/dagmc.cpp

```diff
@@ -303,7 +303,7 @@ void load_dagmc_geometry()
 
     // add to global array and map
     model::surfaces.emplace_back(s);
-    model::surface_map[s->id_] = s->id_;
+    model::surface_map[s->id_] = i;
```

Comment:
presumably ok to start at 0?

Reply:
Might be being picky, but i is an awful long way from its definition, maybe this function should be broken up or something? Not really sure what would make it better?